pelicun/tests/dl_calculation/e1/8000-AIM.json

```diff
@@ -13,16 +13,8 @@
   "assetType": "Buildings",
   "Applications": {
     "DL": {
-      "Application": "Pelicun3",
       "ApplicationData": {
-        "DL_Method": "HAZUS MH EQ IM",
-        "Realizations": 100,
-        "auto_script": "PelicunDefault/Hazus_Earthquake_IM.py",
-        "coupled_EDP": true,
-        "detailed_results": false,
-        "ground_failure": false,
-        "log_file": true,
-        "regional": "true"
+        "ground_failure": false
```

Comment:
I don't think we should remove these settings from the config file. Even though they might be populated with defaults and the simulations run fine, the simulation that runs with defaults is almost surely not the same that would have run with these settings. So, at the end of the day, we are not testing the same example. Let me know your thoughts.

Reply:
I removed them because it appeared to me that they are not used, but I'll have to take a closer look; I might be wrong. If that's the case I'll add them back.

Reply:
Sounds good. I'll keep this conversation open until you get back with more info and we can decide what will happen with those lines.

Reply:
Wait; Seeing this message on the code review page instead of the PR page, I now see that you were referring to more settings, and I realize this is what we also talked about in our last developer meeting. I'll definitely need to make changes regarding this. I'll keep you posted.

Reply:
Wait; seeing this message on the "Files changed" page instead of the "Conversations" page, I now see that you were referring to more settings, and I realize this is what we also talked about in our last developer meeting. I'll definitely need to make changes regarding this. I'll keep you posted.

Reply:
I just figured out why I had removed those entries from the config file. The way I've set up this series of tests is by calling `run_pelicun` directly instead of running `main` via a subprocess.
In the  latest commit, `run_pelicun` has the following arguments:
```
config_path, demand_file, output_path, realizations, detailed_results, coupled_EDP, auto_script_path, custom_model_dir, color_warnings, output_format
```
I moved the relevant entries from the config and passed them in `run_pelicun` directly.

I regret taking this approach, because it doesn't scale well. We should probaby spend some time on a more adaptive approach for testing `DL_calculation.py`, which should also include assertions and repeats to limit stochastic false negatives.
For now, this PR can be merged as is. The tests do in fact execute the intended code.

Reply:
Okay, thanks for confirming. Please create a short issue to capture that improving those tests is on our todo list.

Reply:
I creatred issue #54.